# try,except,else,finally

try:
    print(1/0)
except Exception as error:
    print('erro')
    print(error)
